packages/big-design/src/components/TableNext/Row/useRowState.ts

```diff
@@ -42,23 +42,28 @@ export const useRowState = <T>({
     }
   };
 
-  const hasChildrenRows = childrenRows.length > 0;
+  const hasChildrenRows = childrenRows !== undefined;
```

Comment:
why not `const hasChildrenRows = totalChildrenRows > 0`

Reply:
Good question, this helps to identify when rendering the data dynamically. There could be two scenarios: A row that might have children but at the beginning they are empty because is dynamic, and a row that does not have children at all.  👀  However, I'll recheck again and see how I can better name this ^ 🤔